package.json

```diff
@@ -31,9 +31,16 @@
     "tt-serve-static": "bin/serve-static.js",
     "tt-wdio": "bin/wdio-runner.js"
   },
-  "browserslist": [],
+  "browserslist": [
+    "extends browserslist-config-terra"
+  ],
```

Comment:
Hmm you know, we might not even need this configuration here. Everything within this module is not specific to web development, but are just tools for developing web components. We pass browserslist to the webpack for testing